### PR TITLE
Feature/e UI 5508 work access page display

### PR DESCRIPTION
**Pull request by tarikdogan-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-5508

### Change description ###

On login work access page not displayed for judicial fee-paid-judge user

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/app/components/routing/application-routing.component.spec.ts [hunk 1]
```diff
@@ -1,17 +1,25 @@
 import { of } from 'rxjs';
 import { ApplicationRoutingComponent } from './application-routing.component';
+import {FeatureToggleService} from '@hmcts/rpx-xui-common-lib';
+import { TestBed } from '@angular/core/testing';
```
Comment (connorpgpmcelroy) [inline src/app/components/routing/application-routing.component.spec.ts:4]: Imports incorrectly ordered

File: src/app/components/routing/application-routing.component.spec.ts [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
 
-describe('ApplicationRoutingComponent', () => {
+fdescribe('ApplicationRoutingComponent', () => {
```
Comment (connorpgpmcelroy) [inline src/app/components/routing/application-routing.component.spec.ts:6]: fdescribe

File: src/app/components/routing/application-routing.component.ts [hunk 3]
```diff
@@ -1,18 +1,24 @@
 import { Component, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
 import { select, Store } from '@ngrx/store';
+import { combineLatest, Observable } from 'rxjs';
+import {AppConstants} from '../../../app/app.constants';
 import { WorkAllocationFeatureService } from '../../../work-allocation-2/services';
 import * as fromActions from '../../store';
+import {FeatureToggleService, RoleCategory} from '@hmcts/rpx-xui-common-lib';
+import { map } from 'rxjs/operators';
```
Comment (connorpgpmcelroy) [inline src/app/components/routing/application-routing.component.ts:9]: Incorrect import ordering

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
fdescribe definitely needs replacing and import ordering needs to change

File: src/app/components/routing/application-routing.component.spec.ts (hunk 1 above)
Comment (connorpgpmcelroy) [inline src/app/components/routing/application-routing.component.spec.ts:4]: '@'s should be at top - @angular should be before @hmcts as well

**Review by connorpgpmcelroy — APPROVED** [review]
Approved with minor comment

**Review by udaydenduluri33 — APPROVED** [review]
@tarikdogan-hmcts Thanks for the changes
Please see the functional tests are not broken

**Pull request merged**